.github/workflows/cla.yml

```diff
@@ -15,7 +15,6 @@ jobs:
         uses: contributor-assistant/github-action@v2.2.1
```

Comment:
Let's bump to the latest version please.

Reply:
Done